helm/aws-load-balancer-controller/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: aws-load-balancer-controller
 description: AWS Load Balancer Controller Helm chart for Kubernetes
-version: 1.7.1
-appVersion: v2.7.0
+version: 1.8.0
+appVersion: v2.7.2
```

Comment:
We will bump up the version number during release, please revert this change, thanks.